.github/BOTMETA.yml

```diff
@@ -1391,6 +1393,8 @@ files:
   test/units/modules/network/iosxr: *iosxr
   test/units/modules/network/nso: *nso
   test/units/modules/network/nxos: *nxos
+  test/units/modules/source_control/bitbucket: $team_bitbucket
```

Comment:
Unit tests need moving https://github.com/ansible/ansible/tree/devel/test/units/modules/source_control to match where modules are, ie into `/bitbucket/`

Reply:
> This requires #54599